web_external/models/FolderModel.js

```diff
@@ -0,0 +1,15 @@
+isic.models.FolderModel = girder.models.FolderModel.extend({
+    /**
+     * Remove the contents of the folder.
+     */
+     removeContents: function () {
```

Comment:
Again, I'd rather not confuse our ISIC models with having a Girder thing here. I'd either patch this onto the `girder.models.FolderModel`, or since this code may be more permanent than the `ItemModel.getFiles` make it into a utility method on the View, whichever you think is better.